dev-libs/soci/files/soci-4.0.3-cmake-config-path.patch

```diff
@@ -0,0 +1,27 @@
+# The cmake configuration file provided my upstream is not installed in the
+# appropriate location. This patch moves it to the proper place.
+
+--- a/src/core/CMakeLists.txt
++++ b/src/core/CMakeLists.txt
+@@ -124,7 +124,8 @@ if (SOCI_STATIC)
+     ARCHIVE DESTINATION ${LIBDIR})
+ endif()
+ 
+-install(EXPORT SOCI NAMESPACE SOCI:: DESTINATION cmake)
++install(EXPORT SOCI NAMESPACE SOCI:: DESTINATION
++    ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/cmake)
```

Comment:
```suggestion
+    ${CMAKE_INSTALL_LIBDIR}/cmake)
```

Reply:
I thought i deleted this file.

Upstream is using GNUInstallDirs now, so the patch is no longer needed.